RealmSwift/Realm.swift

```diff
@@ -264,31 +264,94 @@ public final class Realm {
     // MARK: Adding and Creating objects
 
     /**
-     Adds or updates an existing object into the Realm.
+     What to do when an object being added to or created in a Realm has a primary key that already exists.
+     */
+    public enum UpdatePolicy: Int {
```

Comment:
Not sure what the thinking was behind choosing these names, but they differ from the ones in Realm JS (which, granted, were kinda arbitrary):

```
    enum UpdateMode {
        Never = 'never',
        Modified = 'modified',
        All = 'all'
    }
```

Reply:
I should probably change 'changed' to 'modified' to align them.

'error' is trying to convey that an exception is thrown as opposed to it just not updating the object if it already exists (which is a different thing that might be worth doing).